deeplabcut/pose_estimation_pytorch/runners/train.py

```diff
@@ -45,33 +45,43 @@ class TrainingRunner(Runner, Generic[ModelType], metaclass=ABCMeta):
     def __init__(
```

Comment:
I think the docstring of the class is wrong (it is the same as the base Runner's one)

The InferenceRunner's docstring should probably be updated as well